content/gettingStarted/examples/multiPlayerEnv/_index.en.md

```diff
@@ -15,4 +15,14 @@ This example focuses on:
 A dedicated section describing environment settings is presented <a href="../../../envs/#settings">here</a>.
 {{% /notice %}}
 
-{{< github_code "https://raw.githubusercontent.com/diambra/arena/main/examples/multi_player_env.py" >}}
\ No newline at end of file
+{{< github_code "https://raw.githubusercontent.com/diambra/arena/main/examples/multi_player_env.py" >}}
+
+{{% notice note %}}
```

Comment:
The text is pefect. Let's remove the notice shortcode and let's add  a paragraph header: 
### Trained Agent SelfPlay